language/diem-framework/modules/CoreAddresses.move

```diff
@@ -1,50 +1,21 @@
-address 0x1 {
+address DiemFramework = 0x1;
+address Std = 0x1;
```

Comment:
Should this be an undeclared value here, and have its value picked up from the move-stdlib?

Reply:
The stdlib shouldn't assign a value for it, the framework should since it is using the stdlib in its deployment. In other words, move-stdlib should never specify it

Reply:
ahhh very good point! I had it turned around in my head.